src/fdc3/dotnet/DesktopAgent/src/MorganStanley.ComposeUI.DesktopAgent/Contracts/RaiseIntentRequest.cs

```diff
@@ -41,7 +41,7 @@ internal sealed class RaiseIntentRequest
     /// <summary>
     /// Context for identifying more the specific app that should handle the raised intent.
     /// </summary>
-    public Context Context { get; set; }
+    public Context? Context { get; set; }
```

Comment:
raiseIntent request should contain a context object based on the spec: https://fdc3.finos.org/docs/api/ref/DesktopAgent#raiseintent
so in theory it shouldn't be nullable